### PR TITLE
[example] Add face recognition

### LINKED ISSUE #928: [example] Implement Face Recognition example

### PR DESCRIPTION
**Pull request by cuiyanx**
1. Add face recognition example with **facenet** and **face-reidentification** models.
2. Support selecting multiple models.

fixes #928 

Signed-off-by: cuiyan <yanx.cui@intel.com>

### CODE REVIEW

**Comment by cuiyanx** [comment]
@ibelem @BruceDai please review

**Comment by cuiyanx** [comment]
@ibelem please review again, thanks

File: examples/face_recognition/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# WebNN API Face Recognition Example
+This example loads Face Detection models (trained by WIDERFace) and Face Recognition model (trained by davidsandberg) in TensorFlow format, constructs and inferences them by WebNN API.
```
Comment (ibelem) [inline examples/face_recognition/README.md]: Are both Facenet and Face Reidentification Retail models trained by davidsandberg? Are they in TensorFlow format?

Comment (cuiyanx) [reply examples/face_recognition/README.md]: oops! I forgot to update this file, sorry.

**Comment by cuiyanx** [comment]
@ibelem please review

**Comment by ibelem** [comment]
Normalize op was skipped for Facenet, but the Face Reidentification Retail requires PReLU and Activation ops #983 to work for OpenVINO.

Thanks for the great work! Since #982 PReLU and Activation ops were supported in WebGL backend, merge it. Tuning will happen when I have time later than next Tuesday, expect to publish it to github.io within next week.

**Comment by ibelem** [comment]
Error: PReLU is not supported.
    at OpenVINOModelImporter._addOpsAndParams (OpenVINOModelImporter.js:691)
    at OpenVINOModelImporter.createCompiledModel (OpenVINOModelImporter.js:38)
    at async Utils.init (utils.js:73)
    at async utilsInit (main.js:121)
    at async main (main.js:202)

The example still depends on #983 to work.

**Pull request merged**